src/www/js/autoGenerate.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
```

Comment:
This is a very small script, no need to load this over network. Please inline this in the HTML itself